src/Orleans/Streams/Core/StreamSubscriptionHandle.cs

```diff
@@ -13,6 +13,7 @@ public abstract class StreamSubscriptionHandle<T> : IEquatable<StreamSubscriptio
     {
         public abstract IStreamIdentity StreamIdentity { get; }
 
+        public string ProviderName;
         /// <summary>
```

Comment:
@ReubenBond I want to pull the expert here to analyse the situation for me. Adding a string ProviderName to StreamSubscriptionHandle, would this influence backward compatibility of StreamSubscriptionHandle serialization ?